neo/Ledger/Blockchain.cs

```diff
@@ -455,6 +455,7 @@ private void Persist(Block block)
         {
             using (Snapshot snapshot = GetSnapshot())
             {
+                List<ApplicationExecuted> all_application_executed = new List<ApplicationExecuted>();
```

Comment:
I know there already isn’t consistency, but I think it is better if we start adopting camel case instead of underscores in the variable naming convention.